frontend/src/utils/tab.ts

```diff
@@ -0,0 +1,17 @@
+import dayjs from "dayjs";
+import { computed } from "vue";
+import { v1 as uuidv1 } from "uuid";
+import { t } from "../plugins/i18n";
+
+export const defaultTabName = computed(() => t("sql-editor.untitled-sheet"));
```

Comment:
Maybe the tab/sheet name should not have the same situation.
e.g. in TablePlus
<img width="296" alt="image" src="https://user-images.githubusercontent.com/24653555/157811623-006ef607-d62f-4d8c-8165-c8c60682616a.png">
in VSCode
<img width="408" alt="image" src="https://user-images.githubusercontent.com/24653555/157811982-fca97ecc-4cb9-49c9-aedd-d135956ccfa4.png">

And the sheet name is going to be a part of its unique index, so we should name it programmably not a const value.